dangerfile.js

```diff
@@ -15,6 +15,7 @@ changedFiles.forEach((file) => {
   }
 
   const packageName = file.split('packages/')[1].split('/')[0];
+  console.log('file==>', file);
```

Comment:
Do we intend to keep this console.log?

Reply:
No !! will remove this 😄 .

Reply:
Removed : https://github.com/cerner/terra-core/pull/3661/files#diff-5e2c7ef8549eb884675ff159df94c367a4346840514817a626185b400d61b7ca